source/_integrations/fritzbox.markdown

```diff
@@ -29,103 +29,117 @@ ha_integration_type: hub
+The AVM FRITZ!SmartHome integration for Home Assistant allows you to integrate [AVM Smart Home](https://en.avm.de/products/smart-home/) (_former AVM FRITZ!DECT_) devices like plugs, thermostats or shutter drivers as also trigger so called smart home templates (_contains settings for Smart Home devices of the same type_).
 
 #### Tested devices
 
-- [FRITZ!Box 5590 Fiber](https://en.avm.de/products/fritzbox/fritzbox-5590-fiber/)
+- [FRITZ!Box 5590 Fiber][fritzbox_5590_fiber]
 - FRITZ!Box 6490 Cable
-- [FRITZ!Box 6591 Cable](https://en.avm.de/products/fritzbox/fritzbox-6591-cable/)
-- [FRITZ!Box 7590](https://en.avm.de/products/fritzbox/fritzbox-7590/)
-- [FRITZ!Box 7590 AX](https://en.avm.de/products/fritzbox/fritzbox-7590-ax/)
+- [FRITZ!Box 6591 Cable][fritzbox_6591_cable]
+- [FRITZ!Box 7590][fritzbox_7590]
+- [FRITZ!Box 7590 AX][fritzbox_7590_ax]
+- [FRITZ!Box 7530 AX][fritzbox_7530_ax]
 - FRITZ!Box 7490
 - FRITZ!Box 7430
-- [FRITZ!DECT 200](https://en.avm.de/products/fritzdect/fritzdect-200/)
-- [FRITZ!DECT 210](https://en.avm.de/products/fritzdect/fritzdect-210/)
-- [FRITZ!DECT 301](https://en.avm.de/products/fritzdect/fritzdect-301/)
-- [FRITZ!DECT 302](https://en.avm.de/products/fritzdect/fritzdect-302/)
-- [FRITZ!DECT 500](https://en.avm.de/products/fritzdect/fritzdect-500/)
-- [Eurotronic Comet DECT](https://eurotronic.org/produkte/dect-ule-heizkoerperthermostat/comet-dect/)
-- [Magenta SmartHome LED E27 Color](https://www.smarthome.de/geraete/smarthome-led-lampe-e27-farbig-weiss)
+- [FRITZ!DECT 200][fritzdect_200]
+- [FRITZ!DECT 210][fritzdect_210]
+- [FRITZ!DECT 301][fritzdect_301]
+- [FRITZ!DECT 302][fritzdect_302]
+- [FRITZ!DECT 500][fritzdect_500]
+- [Eurotronic Comet DECT][eurotronic_comet_dect]
+- [Magenta SmartHome LED E27 Color][magenta_led_e27_color]
 - Magenta SmartHome LED E27 warmwhite
-- [Rademacher RolloTron DECT 1213](https://www.rademacher.de/shop/rollladen-sonnenschutz/elektrischer-gurtwickler/rollotron-dect-1213)
+- [Rademacher RolloTron DECT 1213][rademacher_rollotron_dect_1213]
 
 {% include integrations/config_flow.md %}
 
 {% note %}
-The configuration in the UI asks for a username. Starting from FRITZ!OS 7.24 the FRITZ!Box creates a random username for the admin user if you didn't set one yourself. This can be found after logging into the FRITZ!Box and visiting System -> FRITZ!Box Users -> Users. The username starts with "fritz" followed by four random numbers. Under properties on the right it says "created automatically". Prior to FRITZ!OS 7.24 the default username was "admin".
+The configuration asks for a username. Starting from FRITZ!OS 7.24 the FRITZ!Box creates a random username for the admin user if you didn't set one yourself. This can be found after logging into the FRITZ!Box and visiting System -> FRITZ!Box Users -> Users. The username starts with "fritz" followed by four random numbers. Under properties on the right it says "created automatically". Prior to FRITZ!OS 7.24 the default username was "admin".
```

Comment:
```suggestion
The configuration asks for a username. Starting from FRITZ!OS 7.24 the FRITZ!Box creates a random username for the admin user if you didn't set one yourself. This can be found after logging into the FRITZ!Box and visiting **System** > **FRITZ!Box Users** > **Users**. The username starts with `fritz` followed by four random numbers. Under **Properties** on the right, it says **created automatically**. Prior to FRITZ!OS 7.24, the default username was `admin`.
```